### PR TITLE
Testing/production

### PR DESCRIPTION
**Pull request by kspeer825**
# Description of change
Add employees and roles to the test_client.
Incorporate production existing tests.

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 version: 2
+
```
Comment (kspeer825) [inline .circleci/config.yml:2]: These lines were accidental, I will remove them.

File: tests/test_client.py [hunk 2]
```diff
@@ -720,6 +770,52 @@ def update_categories(self, obj_id, version):
...
+        uid = self.make_id('employee')[1:]
+        data = {'first_name': uid,
+                'last_name': obj.get('last_name')}
+
+        resp = requests.put(url=full_url, headers=self.get_headers(), json=data)
+        if resp.status_code != 200:
+            raise Exception(resp.text)
+        return resp.json()
+
+    def update_roles_v1(self, obj):
```
Comment (asaf-erlich) [inline tests/test_client.py:793]: This and the above method are nearly identical. Can you create a method they both call? I think it'll save us time in the future if we ever have to make changes to the way we call v1 endpoints.

Comment (kspeer825) [reply tests/test_client.py:793]: I can, should I wait to until settlements is done though? Rule of 3

Comment (kspeer825) [reply tests/test_client.py:793]: Nevermind I'll just do it

File: tests/test_bookmarks.py [hunk 3]
```diff
@@ -200,28 +223,28 @@ def test_run(self):
...
+            first_rec = dict()
+            for message in first_sync_records.get('payments').get('messages'):
+                if message.get('data')['status'] == 'APPROVED':
+                    first_rec = message.get('data')
+                    break
+
+            if not first_rec:
+                raise RuntimeError("Unable to find any any payment with status APPROVED")
+            first_rec_id = first_rec.get('id')
+            first_rec_version = first_rec.get('version')
```
Comment (asaf-erlich) [inline tests/test_bookmarks.py:238]: We do this in another test right? Can we put a method in the base class that they all use?

**Review by asaf-erlich — APPROVED** [review]
made 1 comment, your choice to fix it in this pr or next one

File: tests/test_bookmarks.py (hunk 3 above)
Comment (kspeer825) [reply tests/test_bookmarks.py:238]: We only do updates in this test, I don't think this happens anywhere else.

**Pull request merged**